proposal.tex

```diff
@@ -37,6 +48,17 @@ \subsection{BPS Batch }\label{sec:bpsbatch}
 This would also cover DMS-REQ-0119, DMS-REQ-0125, DMS-REQ-0128, DMS-REQ-0123, DMS-REQ-0127.
 
 
+\subsubsection{Non Rubin code} \label{sec:othercode}
+Users may wish to run arbitrary code on Rubin images.
+It should be possible to provide a butler recipe to get an image or set of images to the users space and allow them to run any code which takes FITS images.
```

Comment:
"butler recipe... and allow them" => "PipelineTask-based shim, using in the BPS system, for iterating over a defined set of image data, extracting the images to user space, and allowing the user"